MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,276 @@
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Validate the specific editor
+
+[ValidateProperty] event allows you to validate specific editor in the data form. You can set `Isvalid`, `ErrorMessage` and `ValidMessage` of [DataFormValidatePropertyEventArgs].
+
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.ValidateProperty += this.OnDataFormValidateProperty;
+
+    private void OnDataFormValidateProperty(object sender, DataFormValidatePropertyEventArgs e)
+    {
+        bool isValid = e.IsValid;
+        string propertyName = e.PropertyName;
+        object newValue = e.NewValue;
+        object currentValue = e.CurrentValue;
+        string errorMessage = e.ErrorMessage;
+        string validMessage = e.ValidMessage;
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Validation label appearance customization
+
+The data form supports to customize the style of both error and valid message label style easily.
+
+#### Customize error label text style
+
+The `error label style` can be customized by changing the [ErrorLabelTextStyle] property of [SfDataForm].
+
+{% tabs %}
+{% highlight XAML %}
+
+    <dataForm:SfDataForm
+        x:Name="dataForm">
+        <dataForm:SfDataForm.ErrorLabelTextStyle>
+            <dataForm:DataFormTextStyle FontSize="10" FontAttributes="Italic" TextColor="Violet" FontFamily="Roboto"/>
+        </dataForm:SfDataForm.ErrorLabelTextStyle>
+    </dataForm:SfDataForm>
+
+{% endhighlight %}
+{% endtabs %}
+
+You can also customize `error label style` for each editor using [ErrorLabelTextStyle] property of [DataFormItem]
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.ErrorLabelTextStyle = new DataFormTextStyle
+                {
+                    TextColor = Colors.DarkSeaGreen,
+                    FontSize = 8,
+                    FontAttributes = FontAttributes.Italic,
+                    FontFamily = "Roboto",
+                };
+            }
+        }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Customize valid message label text style
+
+The `valid message label style` can be customized by changing the [ValidMessageLabelTextStyle] property of [SfDataForm].
+
+{% tabs %}
+{% highlight XAML %}
+
+        <dataForm:SfDataForm
+        x:Name="dataForm">
+        <dataForm:SfDataForm.ValidMessageLabelTextStyle>
+            <dataForm:DataFormTextStyle FontSize="10" FontAttributes="Italic" TextColor="Violet" FontFamily="Roboto"/>
+        </dataForm:SfDataForm.ValidMessageLabelTextStyle>
+    </dataForm:SfDataForm>
+
+{% endhighlight %}
+{% endtabs %}
+
+You can also customize `valid message label style` for each editor using [ValidMessageLabelTextStyle] property of [DataFormItem]
```

Comment:
Also, customize the `valid message label style` for each editor using the [ValidMessageLabelTextStyle] property of the [DataFormItem].